Graphs/VariationTextGraph/index.js

```diff
@@ -198,7 +198,7 @@ export class VariationTextGraph extends AbstractGraph {
                     style={{
                         textAlign: textAlign,
                         cursor: cursor,
-                        fontSize: "1.2em",
+                        fontSize: "1em",
```

Comment:
I increased this on purpose! Any reason you see that this needs to be 1em?

Reply:
Revert the font size change